src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -433,7 +433,15 @@ const formConfig = {
                 type: 'object',
                 properties: {
                   facilityCode: { // TODO: determine whether to store facility ID
-                    type: 'string'
+                    type: 'object',
```

Comment:
I don't think I'm doing this right. The review page isn't showing the `ui:title` anymore. 
![screen shot 2018-08-07 at 15 53 42](https://user-images.githubusercontent.com/4998130/43804743-5b8efda2-9a5a-11e8-8016-0e91a51d7deb.png)